docs/architecture/rewards.md

```diff
@@ -0,0 +1,25 @@
+# Staking Rewards
+
+ZetaChain is a delegated proof of stake blockchain. Holders of ZETA tokens can
+stake their tokens by delegating to validators. Staking tokens allows holders to
+participate in governance and earn rewards for securing the network.
```

Comment:
Should we mention how commissions and slashing might affect these numbers for delegators? 
We use the default distribution module, a reference link to that might work as well

Reply:
393ccbb